Blog post: Testing on Kubernetes with Arquillian Cube

fabiobrz opened the pull request:
SSIA

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+classpath.
+
+If such a definition exists, then Arquillian Cube will apply it to the cluster, and it will provide us with APIs and
+annotations at the class level, that we'll use to wire the test logic up, as we'll see later on.
```
s/level,/level/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+A few changes, provided we started from the previous article about
+https://www.wildfly.org/news/2025/01/27/testing-on-docker-with-cube/[Testing WildFly applications on Docker with Arquillian Cube].
+
+The first thing we need to do is to add a couple more property for two new dependency that we'll need to add, details
```
s/property/properties
s/dependency/dependencies
s/add,/add;/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+        -->
+----
+
+One last thing in the test dependencies section, let's give a version to thr RESTEasy client dependency, since we
```
s/thr/the

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+        </dependency>
+----
+
+Now, onto the (`<build>/<plugins>` section, first off we don't need for the `maven-clean-plugin` to clean up any
```
remove extraneous '('
s/section, first off/section. First off,/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+----
+
+Now, onto the (`<build>/<plugins>` section, first off we don't need for the `maven-clean-plugin` to clean up any
+Docker files, in fact we'll remove those from our project sources later on, since this test will not build nor run any
```
s/files,/files;/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+
+Now, onto the (`<build>/<plugins>` section, first off we don't need for the `maven-clean-plugin` to clean up any
+Docker files, in fact we'll remove those from our project sources later on, since this test will not build nor run any
+Docker images, let's comment the section as follows:
```
s/images, let's/images. Let's/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+
+=== Remove the application sources
+
+Again, we're not building any application here, we rely on a DOcker image on Quay, that contains the application
```
s/here, we/here. We/g
s/DOcker/Docker/g
s/Quay,/Quay/g

bstansberry commented on _posts/2025-02-19-testing-on-k8s-with-cube.adoc:
```diff
@@ -0,0 +1,654 @@
+=== Remove the application sources
+
+Again, we're not building any application here, we rely on a DOcker image on Quay, that contains the application
+already, therefore we don't need the application sources, which can be safely removed:
```
s/already,/already;/g

bstansberry reviewed (changes requested):
I made a bunch of copy edit suggestions.

fabiobrz commented:
> I made a bunch of copy edit suggestions.

Thanks for your review @bstansberry ! Hopefully I addressed your comments (and rebased, hence maybe not so easily visible, sorry about that).
